File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/VideoNoteFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.files.VideoNote;

public class VideoNoteFilterValidator extends FilterValidator<VideoNote> {

    public VideoNoteFilterValidator(CustomValidator<VideoNote> validator) {
        super(VideoNote.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/VideoNoteFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.std.files.VideoNote
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class VideoNoteFilterValidator(validator: Validator<VideoNote>) : FilterValidator<VideoNote>(
    VideoNote::class, validator)

